src/main.rs

```diff
@@ -101,8 +104,13 @@ pub async fn dm_blocks(user_id: String, blocks: Vec<Value>) -> Result<(), String
     Ok(())
 }
 
-async fn gift_dm(giver: &str, new_owner: &str, possession: &Possession, count: usize) -> Result<(), String> {
-    dm_blocks(new_owner.to_string(), {
+async fn gift_dm(
+    giver: &str,
+    new_owner: &str,
+    possession: &Possession,
+    count: usize,
+) -> Result<(), String> {
+    dm_blocks(new_owner.to_string(), "You've received a gift!".to_string(), {
```

Comment:
Five "You've received a gift!" notifications are almost as useless as the current "this can't be displayed" notifications.

Reply:
Giving this another thought, the purpose is just to get them to look at what actually happened, so maybe this suffices.

Reply:
Yep, my goal with the notifications was to give a basic idea of what happened and getting them to open the app - most stuff should be done from app home / DM blocks imo.